glean/src/core/events/index.ts

```diff
@@ -12,53 +12,57 @@ export class CoreEvent<
    Context extends unknown[] = unknown[],
    // The expected type of the action result. To be returned by the plugin.
    Result extends unknown = unknown
- > {
-   // The plugin to be triggered eveytime this even occurs.
-   private plugin?: Plugin<CoreEvent<Context, Result>>;
+> {
+  // The plugin to be triggered eveytime this even occurs.
```

Comment:
Since we are here...

```suggestion
  // The plugin to be triggered eveytime this event occurs.
```